packages/icons/src/HarnessIcons.ts

```diff
@@ -1302,6 +1302,7 @@ import StatsUpSquare from './stats-up-square.svg'
 import StatusPending from './status-pending.svg'
 import StatusRunning from './status-running.svg'
 import StatusUpgrade from './status-upgrade.svg'
+import Statuspage from './statuspage.svg'
```

Comment:
Please rename it to `status-page`

```suggestion
import Statuspage from './status-page.svg'
```

Reply:
Please make the changes wherever mentioned